rabbitmq-operator/src/main/java/com/indeed/operators/rabbitmq/api/RabbitManagementApiFacade.java

```diff
@@ -0,0 +1,370 @@
+        return executeCall(() -> api.createOperatorPolicy(vhost, name, policy));
+    }
+
+    public ResponseBody deleteOperatorPolicy(final String vhost, final String name) {
+        return executeCall(() -> api.deleteOperatorPolicy(vhost, name));
+    }
+
+    public List<Queue> listQueues() {
+        return executeCall(() -> api.listQueues());
+    }
+
+    public List<Queue> listQueues(final String vhost) {
+        return executeCall(() -> api.listQueues(vhost));
+    }
+
+    public Queue getQueue(final String vhost, final String name) {
+        return executeCall(() -> api.getQueue(vhost, name));
+    }
+
+    public ResponseBody createQueue(final String vhost, final String name, final Queue queue) {
+        return executeCall(() -> api.createQueue(vhost, name, queue));
+    }
+
+    public ResponseBody deleteQueue(final String vhost, final String name) {
+        return executeCall(() -> api.deleteQueue(vhost, name));
+    }
+
+    public ResponseBody purgeQueue(final String vhost, final String name) {
+        return executeCall(() -> api.purgeQueue(vhost, name));
+    }
+
+    public List<User> listUsers() {
+        return executeCall(() -> api.listUsers());
+    }
+
+    public User getUser(final String name) {
+        return executeCall(() -> api.getUser(name));
+    }
+
+    public ResponseBody createUser(final String name, final User user) {
+        return executeCall(() -> api.createUser(name, user));
+    }
+
+    public ResponseBody deleteUser(final String name) {
+        return executeCall(() -> api.deleteUser(name));
+    }
+
+    public User whoami() {
+        return executeCall(() -> api.whoami());
+    }
+
+    public List<VirtualHost> listVirtualHosts() {
+        return executeCall(() -> api.listVirtualHosts());
+    }
+
+    public VirtualHost getVirtualHost(final String vhost) {
+        return executeCall(() -> api.getVirtualHost(vhost));
+    }
+
+    public ResponseBody createVirtualHost(final String vhost) {
+        return executeCall(() -> api.createVirtualHost(vhost));
+    }
+
+    public ResponseBody deleteVirtualHost(final String vhost) {
+        return executeCall(() -> api.deleteVirtualHost(vhost));
+    }
+
+    public List<Shovel> listShovels() {
+        return executeCall(() -> api.listShovels());
+    }
+
+    public List<Shovel> listShovels(final String vhost) {
+        return executeCall(() -> api.listShovels(vhost));
+    }
+
+    public Shovel getShovel(final String vhost, final String name) {
+        return executeCall(() -> api.getShovel(vhost, name));
+    }
+
+    public ResponseBody createShovel(final String vhost, final String name, final Shovel shovel) {
+        return executeCall(() -> api.createShovel(vhost, name, shovel));
+    }
+
+    public ResponseBody deleteShovel(final String vhost, final String name) {
+        return executeCall(() -> api.deleteShovel(vhost, name));
+    }
+
+    private static <T> T executeCall(Supplier<Call<T>> f) {
```

Comment:
final here, for consistency.  You could sprinkle some finals in the function body if you wanted as well.